segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AbstractConnectionListener.java

```diff
@@ -159,6 +202,11 @@ public void initChannel(SocketChannel ch) {
 
         // Start the server.
         serverChannel = b.bind(host, port).awaitUninterruptibly().channel();
+
+        if (healthServiceManager != null) {
+            healthServiceManager.register(new ConnectionListenerHealthContributor(type,
```

Comment:
This constructor looks super redundant. The `listener` (i.e., the second argument that you pass `this`) object already contains the `host`, `port`, and not to mention the class. So, just writing `new ConnectionListenerHealthContributor(this)` may be enough, there is no need for all the other arguments, right?